en/admin/db_manage.rst

```diff
@@ -57,9 +57,9 @@ The volumes of CUBRID database are classified as permanent volume, temporary vol
 *   In the permanent volumes,
 
     *   there are data volumes, that usually store permanent data, but can also store temporary data.
-    *   there are log volumes, that can be further classified as: one active log, archive logs and one background archiving log.
+    *   there is the log volume, that can be further classified as: one active log.
```

Comment:
Why does it need to 'classify'? This just says 'one active log'.

Reply:
How about this fix?